README.md

```diff
@@ -1,20 +1,22 @@
 [![CircleCI](https://circleci.com/gh/bids-standard/bids-website.svg?style=shield)](https://circleci.com/gh/bids-standard/bids-website)
 
+<img src="https://github.com/bids-standard/bids-specification/blob/master/BIDS_logo/BIDS_logo_black_transparent_background_crop.png?raw=true" alt="bids-logo" width="600"/>
+
 # BIDS website
 
 This is the repository for the Brain Imaging Data Structure (BIDS) website that is hosted at <https://bids.neuroimaging.io>.
 
 ## Contributing to the BIDS website
 
-The website is hosted [here](https://bids.neuroimaging.io). To contribute to the website, please submit an [issue](https://github.com/bids-standard/bids-website/issues) or send a [pull request](https://github.com/bids-standard/bids-website/pulls) in this repository.
+To contribute to the website, please submit an [issue](https://github.com/bids-standard/bids-website/issues) or send a [pull request](https://github.com/bids-standard/bids-website/pulls) to this repository.
 
 The more static website pages are located in `_pages`, the blog posts that are listed in a reverse chronological order (i.e., the newest content on top) are located in `_posts`.
 
 Please see the `docs` directory for the technical documentation for this website.
 
 ## Contributing to the BIDS specification
 
-The specification is hosted [here](http://bids-specification.readthedocs.io/). To contribute to the specification, please submit an [issue](https://github.com/bids-standard/bids-specification/issues) or send a [pull request](https://github.com/bids-standard/bids-specification/pulls) on the [bids-specification](https://github.com/bids-standard/bids-specification) repository.
+The specification is hosted on [https://bids-specification.rtfd.io](https://bids-specification.rtfd.io). To contribute to the specification, please submit an [issue](https://github.com/bids-standard/bids-specification/issues) or send a [pull request](https://github.com/bids-standard/bids-specification/pulls) on the [bids-specification](https://github.com/bids-standard/bids-specification) repository.
```

Comment:
```suggestion
The specification is rendered at [https://bids-specification.readthedocs.io/en/stable/](https://bids-specification.readthedocs.io/en/stable/) by [ReadTheDocs](https://readthedocs.org/). To contribute to the specification, please submit an [issue](https://github.com/bids-standard/bids-specification/issues) or send a [pull request](https://github.com/bids-standard/bids-specification/pulls) on the [bids-specification](https://github.com/bids-standard/bids-specification) repository.
```

It appears both links direct to the same URL (https://bids-specification.readthedocs.io/en/stable/) so likely doesn't matter which link we use. I thought it would be nice to give credit to RTD for rendering our docs but not needed

Reply:
```suggestion
The specification is hosted on [https://bids-specification.readthedocs.io](https://bids-specification.rtfd.io). To contribute to the specification, please submit an [issue](https://github.com/bids-standard/bids-specification/issues) or send a [pull request](https://github.com/bids-standard/bids-specification/pulls) on the [bids-specification](https://github.com/bids-standard/bids-specification) repository.
```

Reply:
fair point, thanks @franklin-feingold but I only changed `rtfd` to `readthedocs` in the displayed link and left everything else as it was